types/auth0-lock/index.d.ts

```diff
@@ -96,6 +96,7 @@ interface Auth0LockAuthParamsOptions {
 }
 
 interface Auth0LockAuthOptions {
+    autoParseHash?: boolean;
```

Comment:
Please add a test of this property being used in order to prevent regressions.

Reply:
@carusology  options have now been added to the test. I set autoParseHash to true in the authentication options since we are not calling resumeAuth in the tests. Hope this is ok.